honeycomb-core/src/lib.rs

```diff
@@ -11,6 +11,7 @@
 
 pub mod dart;
 pub mod embed;
+pub mod generation;
```

Comment:
Not sure if it will stay there. Looks more an external utility to me.

Reply:
Probably, it could be in the lib associated to the crate containing benchmarks.